examples/java/spring-boot/manifest.yml

```diff
@@ -0,0 +1,7 @@
+---
+applications:
+  - name: federated-api-model-spring-boot-sandbox
+    buildpacks:
+      - https://github.com/cloudfoundry/java-buildpack.git
```

Comment:
What can we do to version pin here? It'd be best if we can restrict this to a specific version of the buildpack to ensure we don't get hit by big tweaks in the future i.e. new required environment variables